R/html_document.R

```diff
@@ -612,7 +637,9 @@ html_document <- function(toc = FALSE,
     on_exit = on_exit,
     base_format = html_document_base(theme = theme,
                                      self_contained = self_contained,
-                                     lib_dir = lib_dir, mathjax = mathjax,
+                                     lib_dir = lib_dir,
+                                     math = math,
+                                     mathjax = mathjax,
```

Comment:
I need to reopen the discussion about this option name. I was not expecting this but it is causing issue because of R partial argument matching in functions. 
This is creating issues with other format that extend this one : 

Example of behavior
```r
# base format defines a new argument math
base_format <- function(math = "katex", mathjax = "default"){ math } 
# expected
base_format(math = "dummy")
#> [1] "dummy"

# another function use this format not knowing explicitly the new argument
out_format <- function(mathjax = "default", ...) { base_format(mathjax = mathjax, ...)}
# Not expected - mathjax is matched because `...` is after `mathjax`
out_format(math = "dummy")
#> [1] "katex"
```

Changing the argument name would solve this - or it would require to explictly set the argument in all extension format. It seems not right to keep that IMO.

Reply:
Argument name was changed to `math_method`